boards/posix/nrf52_bsim/hw_models_version

```diff
@@ -1 +1 @@
-v1.8
+test_2019_11_08_CIc
```

Comment:
@aescolar @anangl can this actually be merged as is? Just double-checking

Reply:
Yes, this is fine, we checked with @aescolar offline

Reply:
oh great

Reply:
I will send a trivial fix right after this is merged updating it to v2.0 (they both point to the same sha)